web-registry/src/hooks/useBasketTokens.ts

```diff
@@ -0,0 +1,46 @@
+import { useState, useEffect } from 'react';
+
+import { Basket } from '@regen-network/api/lib/generated/regen/ecocredit/basket/v1/types';
+import { QueryBasketsResponse } from '@regen-network/api/lib/generated/regen/ecocredit/basket/v1/query';
+import { QueryBalanceResponse } from '@regen-network/api/lib/generated/cosmos/bank/v1beta1/query';
+
+import useQueryBaskets from './useQueryBaskets';
+import useQueryBalance from './useQueryBalance';
+
+export interface BasketTokens {
+  basket: Basket;
+  balance?: QueryBalanceResponse | undefined;
+  displayDenom: string;
+}
+
+export default function useBasketTokens(
```

Comment:
I strongly agree with the module level index. In fact, I was going to do it, but I still have some doubts about the structure of this hooks module... Not so much, or I'm still not sure, with not exporting as default the component or code that corresponds to the file as 1 to 1. Anyway, we will deal with it soon. For now, since the hooks directory is new, I'm going to leave it with default export for each hook, and add the index file to export at the module level. Thanks!!

```
export { default as useBasketDetails } from './useBasketDetails';
export { default as useBasketTokens } from './useBasketTokens';
export { default as useEcocredits } from './useEcocredits';
```